content/packages-and-modules/updating-and-managing-your-published-packages/unpublishing-packages-from-the-registry.mdx

```diff
@@ -5,7 +5,7 @@ import shared from '../../../src/shared.js'
 
 ## How to unpublish
 
-As a package owner or collaborator, if your package has no dependents, you can permanently remove it from the npm registry by using the CLI. You can [unpublish][unpublish-cli] within 72 hours of the initial publish. Beyond 72 hours,so you can still unpublish your package if [it meets certain criteria](https://www.npmjs.com/policies/unpublish).
+As a package owner or collaborator, if your package has no dependents, you can permanently remove it from the npm registry by using the CLI. You can [unpublish][unpublish-cli] within 72 hours of the initial publish. Beyond 72 hours,so you can still unpublish your package if [it meets certain criteria][unpublish].
```

Comment:
This (link) definitely works? Doesn't need a leading `/` ?

Reply:
Or it's a reference to [this](https://github.com/npm/documentation/pull/20/files#diff-5176d1ec8ff171066d535aca53f76a8aeee6c17f0bc924703df9c05089d93c32R68) ?

Reply:
Fixed on one of the links - however, I am not sure where the [unpublish-cli] should actually point to. Any ideas?